<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="IBM-HWQV2F3" failures="2" tests="2" name="Default test" time="29.076" errors="0" timestamp="2024-05-19T15:08:06 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="15.874">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at org.testng.AssertJUnit.fail(AssertJUnit.java:65)
at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:23)
at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)
at stepDefinitions.stepDefinitionImpl.verify_successMsg_is_displayed(stepDefinitionImpl.java:62)
at ✽.I verify "THANK YOU FOR THE ORDER." is displayed(file:///C:/Users/003OPW744/eclipse-workspace/FrameworkDesign/src/test/java/cucumber/PurchaseOrder.feature:11)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="12.693">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at stepDefinitions.stepDefinitionImpl.submit_the_order_for_product(stepDefinitionImpl.java:52)
at ✽.submit the order for ADIDAS ORIGINAL(file:///C:/Users/003OPW744/eclipse-workspace/FrameworkDesign/src/test/java/cucumber/PurchaseOrder.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
